.github/workflows/docker-ort-runtime-ext.yml

```diff
@@ -19,14 +19,14 @@ name: Mega Docker extended image
 
 on:
   workflow_dispatch:
+  schedule:
+    - cron: '0 4 * * *'
   pull_request:
     paths:
       - '.versions'
       - 'Dockerfile'
       - 'Dockerfile-extended'
   push:
-    branches:
-      - main
     tags:
```

Comment:
Why can the main branch be removed? (Guess hotfix releases?)

Reply:
Now push any tag. Tags can be in any branch you made it.
Before this push when tagged and when is pushed to main, not two at same time. They are individual entities.